src/calibrate_all.jl

```diff
@@ -53,6 +107,15 @@ function calibrate_all(data::LegendData, sel::AnyValiditySelection, datastore::A
     trig_e_cusp_ctc_cal = _fix_vov(getindex.(ged_events_pre.e_cusp_ctc_cal, trig_e_ch))
     trig_e_535_cal      = _fix_vov(getindex.(ged_events_pre.e_535_cal, trig_e_ch))
     trig_t0 = _fix_vov(getindex.(ged_events_pre.t0, trig_e_ch))
+
+    # Check if any events have only NaN t0 values (detectors without calibration)
+    # Note: Use ustrip for Unitful compatibility
+    n_events_all_nan_t0 = count(t0 -> all(x -> isnan(ustrip(x)), t0), trig_t0)
```

Comment:
No need to ustrip when checking for `NaN`:

<img width="302" height="61" alt="Image" src="https://github.com/user-attachments/assets/36b8fd42-bfed-401d-a8f8-f67a23623b35" />


```suggestion
    n_events_all_nan_t0 = count(all.(isnan, trig_t0))
```